converters/fromZigbee.js

```diff
@@ -5854,6 +5854,35 @@ const converters = {
             }
         },
     },
+    legrand_device_mode: {
+        cluster: 'manuSpecificLegrandDevices',
+        type: ['readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {};
+            const option0 = msg.data['0'];
+            // Beware that mode depends on device type
+            switch (option0) {
+            // contactor
+            case 0x0003:
+                payload.device_mode = 'switch';
+                break;
+            case 0x0004:
+                payload.device_mode = 'auto';
+                break;
+
+            // dimmer
+            case 0x0101: payload.device_mode = 'dimmerOn';
+                break;
+            case 0x0100: payload.device_mode = 'dimmerOff';
```

Comment:
Make this snake case, dimmerOn -> dimmer_on, dimmerOff -> dimmer_off

Reply:
I patched according your suggestions + fixed a typo